src/frontend/komponenter/Fagsak/Vedtak/FeilutbetaltValuta/FeilutbetaltValutaPeriode.tsx

```diff
@@ -71,16 +72,18 @@ const FeilutbetaltValutaPeriode: React.FC<IFeilutbetaltValutaPeriode> = ({
             onOpenChange={håndterLukkingOgÅpningAvPanel}
             content={
                 <FlexColumnDiv>
-                    <FeilutbetaltValutaSkjema skjema={skjema} />
+                    {erRadEkspandert && <FeilutbetaltValutaSkjema skjema={skjema} />}
```

Comment:
Må ha denne for å trigge ny render av skjemaet ved åpning lukking (og dette er for å sikre at datovelgeren rendrer riktig verdi). Litt hacky løsning 🙃